addons/mail/models/discuss/discuss_channel.py

```diff
@@ -510,12 +510,14 @@ def add_members(
         self, partner_ids=None, guest_ids=None, invite_to_rtc_call=False, post_joined_message=True
     ):
         """ Adds the given partner_ids and guest_ids as member of self channels. """
-        return self._add_members(
+        all_new_members = self._add_members(
             partners=self.env["res.partner"].browse(partner_ids or []).exists(),
             guests=self.env["mail.guest"].browse(guest_ids or []).exists(),
             invite_to_rtc_call=invite_to_rtc_call,
             post_joined_message=post_joined_message,
         )
+        self._post_add_members_hook(all_new_members)
```

Comment:
We can probably just override add_members directly?

Reply:
And probably the private one to get consistent results.

Reply:
All base methods are basically hooking points, we don't necessarily create specific methods for them.
Only when it must be some place "inside", but if it's fully "after" then we usually don't.